generator/Makefile

```diff
@@ -81,7 +85,7 @@ docker-tag-latest:
 	docker tag "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME):latest"
 
 .PHONY: mibs
-mibs: mib-dir \
+mibs: clean mib-dir \
```

Comment:
Hmm, I'm not sure I would want to always `make clean` on every call to `make mibs`.

Reply:
The problem is that if you have an "old" mibs dir already in, running `make mibs` does nothing.

Reply:
Yes, that's intentional. It's designed to be additive-only. Calling `make clean` should be something you do intentionally.

Reply:
Ok, removed it.